### ISSUE TITLE
[Common] Add extra sync features such as Dropbox

### ISSUE THREAD

**Issue by gdavis** [gdavis/Mimir-Feedback#49]
Users should be able to use other file syncing platforms to sync the library file in Mimir. Look into other available SDKs to support this.

**Comment by svranyes** [comment]
if there were a way to sync the podcast that is playing and the queue that would be ideal. The scenario that is important for me is handing off from one device to the next - e.g. phone in the car to my mac when I get to the office or the inverse.

**Comment by gdavis** [comment]
@svranyes Absolutely. A lot of this is already in place, but I have a bug that sometimes devices with other data overwrites what is in iCloud. When its all fixed up, we'll have fully synced play positions and playlist.

**Comment by gdavis** [comment]
Currently not planning on implementing this feature
